iroh-api/src/p2p.rs

```diff
@@ -69,6 +71,13 @@ impl P2p for ClientP2p {
         }
         .map_err(|e| map_service_error("p2p", e))
     }
+
+    async fn peers(&self) -> Result<HashMap<PeerId, Vec<Multiaddr>>> {
```

Comment:
I prefer BTreeMap, less random...

Reply:
interesting. I'll make a note in a follow-up issue